Tests/ComposableArchitectureTests/LocalizedStringKeyTests.swift

```diff
@@ -11,6 +11,7 @@ class LocalizedStringKeyTests: XCTestCase {
     )
 
     let formatter = NumberFormatter()
+    formatter.locale = Locale(identifier: "en_US")
```

Comment:
Nice change. You could even consider using `"en_US_POSIX"`, which is guaranteed not to change in the future. (Not that it's a big deal if the test breaks in the distant future if the US ever changes its number format.)